apps/api/src/schema/websoc.ts

```diff
@@ -118,30 +136,73 @@ export const websocQuerySchema = z.object({
       }
       return parsedNums;
     }),
-  instructorName: z.string().optional(),
-  days: daysSchema.optional(),
-  building: z.string().optional(),
-  room: z.string().optional(),
+  instructorName: z.string().optional().openapi({
+    description: "The instructor's name of the WebSoc section",
+    example: "Anteater, P.",
+  }),
+  days: daysSchema.optional().openapi({
+    description: "Selects WebSoc sections held on any of the given days",
+    example: "M,Tu,W,Th",
```

Comment:
If convenient, consider demonstrating more day aliases as defined in the day schema. My previous comment about expressly articulating comma separation stands.